source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRTextBlockRenderer.mm

```diff
@@ -41,6 +41,11 @@ - (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup
     std::shared_ptr<HostConfig> config = [acoConfig getHostConfig];
     std::shared_ptr<BaseCardElement> elem = [acoElem element];
     std::shared_ptr<TextBlock> txtBlck = std::dynamic_pointer_cast<TextBlock>(elem);
+
+    if (txtBlck->GetText().empty()) {
+        return nil;
+    }
```

Comment:
We should add a sample test card that reflects this behavior as part of this work. @jwoo-msft

Reply:
Sure, I will add sample test card update with UWP changes.